src/compute-client/src/controller/instance.rs

```diff
@@ -361,12 +371,23 @@ where
 
         // Add the replica
         self.compute
-            .add_replica(id, addrs, logging_config, communication_config);
+            .add_replica(id, config.location, logging_config);
         Ok(())
     }
 
-    /// Remove an existing instance replica, by ID.
+    /// Remove an existing instance replica, by ID. Will remove the replica from the
+    /// orchestrator.
     pub async fn remove_replica(&mut self, id: ReplicaId) -> Result<(), ComputeError> {
+        if let Err(e) = self.compute.replicas[&id].send(ComputeCommand::DropInstance) {
+            tracing::warn!("Could not send DropInstance to replica {:?}: {}", &id, &e)
```

Comment:
Since the orchestrator removal hinges on our ability to send `DropInstance` to the replica task, do you think it would make sense to elevate the logging level to error here? IMO, whenever we see this in the logs, we need to assume that we just leaked a computed pod, so we should go and clean it up manually. Maybe that would also be a good thing to add to the log message.